src/main/java/org/betonquest/betonquest/menu/MenuItem.java

```diff
@@ -364,4 +327,42 @@ private Map<String, ItemDescription> generateLanguageDescriptions(final String m
 
         return descriptions;
     }
+
+    /**
+     * Extended, static copy of org.betonquest.betonquest.Instruction.Item for easier quest item handling
+     */
+    @SuppressWarnings("PMD.ShortClassName")
+    public static class Item {
```

Comment:
Could this be a `record`?

Reply:
This was only moved by our formatter, so this is not related to my PR